frontend/src/components/common/NotificationStack.vue

```diff
@@ -1,26 +1,71 @@
 <script setup lang="ts">
-import { notifications } from '@/store/NotificationState';
+import {
+    notifications,
+    showFullNotificationMsg,
+} from '@/store/NotificationState';
+import { IconSize } from '@/enum/IconEnum';
+import Message from 'primevue/message';
 </script>
 
 <template>
-    <template v-for="(value, key) in notifications" :key="key">
-        <NotificationMessage
-            v-if="value.msg !== ''"
-            :severity="key.toString()"
-            :msg-text="value.msg"
-            :hasFullMsg="value.fullMsg !== ''"
-            class="notification-stack"
+    <Message
```

Comment:
Hi Jota, just curious, do we need to change the notification message for this ticket?

Reply:
Refactoring and removing the need for the v-if condition from the notification message restores the PrimeVue animation.

Reply:
Hi Jota, sorry maybe I miss understood, which animation we're trying to get here?

Reply:
The animation on the message leave event, I restored the primevue animation but I noticed that was not working ok, so I added a transition-group. should be working as expected now

Reply:
Hi Jota, could you please capture a screenshot for me about the animation? thanks!! or maybe you can show me when we chat today, I just want to avoid unnecessary changes for the notification part. thanks!!